tests/main/install-sideload/task.yaml

```diff
@@ -70,6 +70,9 @@ execute: |
     snap install --dangerous ./test-snapd-tools_1.0_all.snap
     test-snapd-tools.success
 
+    echo "All snap blobs are 0600"
+    test "$(find /var/lib/snapd/ \( -name \*.snap -o -path /var/lib/snapd/cache/\* \) -printf '%#m\n' | sort -u | xargs)" = "0600"
```

Comment:
I think you could simplify the find expression to just:
`test "$(find /var/lib/snapd/{cache,snaps} -type f -printf '%#m\n' | sort ....`
and get same result (seems to work here).

Reply:
To catch seeds we would need `find /var/lib/snapd/{cache,snaps,seed/snaps}....` obviously... This is if we want to be more explicit about what are we checking... Anyway, just a nitpick.